bin/rtlcss.js

```diff
@@ -85,7 +85,15 @@ while ((arg = args.shift())) {
       break
     case '-s':
     case '--silent':
-      console.log = console.info = console.warn = function () {}
+      console.info = console.warn = function () {}
+      break
+    case '-x':
+    case '--exclude':
+      exclude = args.shift().split(',').map(function(str) {
```

Comment:
Actually i'm not sure about this one, in my case, only a few files need to be excluded :)